### PR TITLE
Datetime

### LINKED ISSUE #2245: Date issue
- While executing an order
- While generating an invoice
Try to execute the order and generate an invoice at 10:55 pm
In the client, the panel date shows correctly
image
In the admin panel, it's showing incorrect
image
Generated invoice date 
image

### PR DESCRIPTION
**Pull request by Gurmeenkour99**
fixes #2245 #2360 #2438

### CODE REVIEW

File: app/Http/Controllers/Front/BaseClientController.php [hunk 1]
```diff
@@ -170,6 +170,7 @@ public function postPassword(ProfileRequest $request)
 
     public function getInvoicesByOrderId($orderid, $userid, $admin = null)
     {
+        dd('jhgfcx');
```
Comment (sowmiyasubramaniyan) [inline app/Http/Controllers/Front/BaseClientController.php]: remove dd

**Review by sowmiyasubramaniyan — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Front/BaseClientController.php (hunk 1 above)
Comment (Gurmeenkour99) [reply app/Http/Controllers/Front/BaseClientController.php]: pushed

**Review by sowmiyasubramaniyan — DISMISSED** [review]

**Review by sandesh556 — APPROVED** [review]

**Comment by SarfrazS** [comment]
@Gurmeenkour99 Please resolve conflicts and let me know.

**Comment by Gurmeenkour99** [comment]
Conflicts has been resolved

**Comment by DhanyaChengappa** [comment]
@Sabaric12 Update these details:

**First Round Test summary**
Total time in days:

Total issues:
Minor -
Major -
critical -

**Second Round Test summary**
Total time in days:

Total issues:
Minor -
Major -
critical -

**Comment by Sabaric12** [comment]
Tested this pr with the existing installation on server instance 
The Mentioned Issues are working fine now 
With The Admin Login with the Pacific/Midway time zone now I purchased the product the time is coming according to the Pacific/Midway time zone only
image
I Tested the user with the Australia/Melbourne Timezone and Asia/Kolkatta Timezone
image
First Round Test summary
Total time in days:5 Hours
Total issues:0
Minor -
Major -
critical -

**Review by Sabaric12 — APPROVED** [review]

**Review by SarfrazS — APPROVED** [review]
Last commits
image
Re-tested on server instance in below places with different time zone. Working fine.

1. Invoice popup Page.
2. Order data table page
3. Invoice data table page.
4. Order view page
5. Invoice view page.

Second Round Test Summary
Total time in days:1 Hours
Total issues:0
Minor -
Major -
critical -

Note:- Based on the user's time zone, time is showing for him.

**Pull request merged**